src/components/BatchWithdrawFlow/flow/SelectBatchesWithdrawnQuantity.tsx

```diff
@@ -72,18 +72,21 @@ const defaultTableColumns = (): TableColumnType[] => [
     key: 'germinatingQuantityWithdrawn',
     name: strings.GERMINATING_QUANTITY,
     type: 'string',
+    alignment: 'right',
```

Comment:
Looks like we are doing this for all number type columns.
Should we fix the `type` in all the places as `number`, under common/table if the column type is a number we can update the columns to set alignment to right? Just a thought. It will work for all future columns as well as long as we set the type to number correctly.

Reply:
I thought about it, but wasn't sure if setting type number would change anything at tables, but I was checking the code and it looks like it doesn't, so I will try this approach, sounds more correct!

Reply:
To be honest, I don't know if the other approach has advantages. Either way we need to ensure setting a column property correctly in the future (type as number or alignment as right). Maybe we should just stick with your current approach. The other suggestion is not justified.

Reply:
In case we have other columns of type number, we would want them to be right aligned, so I think it's correct to do it like that